standalone/assembly-allowlist.txt

```diff
@@ -299,6 +299,7 @@
 /mule-standalone-${productVersion}/lib/opt/opentelemetry-sdk-metrics-1.10.1-alpha.jar
 /mule-standalone-${productVersion}/lib/opt/opentelemetry-sdk-trace-1.10.1.jar
 /mule-standalone-${productVersion}/lib/opt/opentelemetry-semconv-1.10.1-alpha.jar
+/mule-standalone-${productVersion}/lib/opt/mule-sdk-compatibility-api-1.0.0-SNAPSHOT.jar
```

Comment:
is this enough? don't you also need to add `<include>` and `<exclude>` elements in the `assembly.xml`?

Reply:
fixed